samples/metrics_monitor/CMakeLists.txt

```diff
@@ -35,11 +35,16 @@ if( PKG_LIBDRM_FOUND AND
     PKG_PCIACCESS_FOUND AND
     BUILD_TESTS)
 
-    file( GLOB_RECURSE test_srcs "${CMAKE_CURRENT_SOURCE_DIR}/test/*.cpp" )
+    # test_monitor sample
 
-    add_executable( test_monitor ${test_srcs})
-    target_include_directories( test_monitor PRIVATE ./include ${PKG_LIBDRM_INCLUDE_DIRS})
-    target_link_libraries( test_monitor pthread rt gtest cttmetrics ${PKG_LIBDRM_LIBRARIES})
+    set( sources "" )
+
+    file( GLOB_RECURSE test_srcs "${CMAKE_CURRENT_SOURCE_DIR}/test/*.cpp" "${CMAKE_CURRENT_SOURCE_DIR}/igtload/*.c" )
```

Comment:
Can you avoid creating new directory called igtload? Place all files in test, please.

Reply:
fixed